debugtools/DDR_VM/src/com/ibm/j9ddr/vm29/tools/ddrinteractive/gccheck/CheckEngine.java

```diff
@@ -102,6 +102,7 @@ class CheckEngine
 	private int _ownableSynchronizerObjectCountOnList = UNINITIALIZED_SIZE; /**< the count of ownableSynchronizerObjects on the ownableSynchronizerLists, =UNINITIALIZED_SIZE indicates that the count has not been calculated */
 	private int _ownableSynchronizerObjectCountOnHeap = UNINITIALIZED_SIZE; /**< the count of ownableSynchronizerObjects on the heap, =UNINITIALIZED_SIZE indicates that the count has not been calculated */
 	private boolean _needVerifyOwnableSynchronizerConsistency = false;
+	private boolean _isVirtualLargeObjectHeapEnabled = false;
```

Comment:
Please don't explicitly initial fields to their default values.

Reply:
This has not been addressed; if you like, the initializer for `_needVerifyOwnableSynchronizerConsistency` could be removed as well.